atom_gems/AtomTutorials/Code/Platform/Android/PAL_android.cmake

```diff
@@ -0,0 +1,4 @@
+
+set(PAL_TRAIT_ATOMTUTORIALS_SUPPORTED TRUE)
```

Comment:
Since we don't actually have any platform specific traits or code, we should be able to remove this whole folder. It looks like the Atom/RPI gem eschews the whole platforms folder, and in its .CMakeLists.txt file it uses more generic 'PAL_TRAIT_BUILD_TESTS_SUPPORTED' and 'PAL_TRAIT_BUILD_HOST_TOOLS' instead of traits specific to the gem, so we can probably do the same.

It looks like it will just be a matter of removing these traits from the CMakeLists.txt file.

Reply:
Makes sense, I deleted the Platform folder & edited CMakeLists to delete all the AtomTutorials PAL traits and dependence on the platform files.